doc/ngsi/docs/SchemasAnalyzer.md

```diff
@@ -1,34 +1,35 @@
 	* [Importing several schemas from different resources](#importing-several-schemas-from-different-reources)
 * [Using the Data Models Schemas for verifying the structure of a data model](#using-the-data-models-schemas-for-verifying-the-structure-of-a-data-model)
-	* [Verifying an entity with a JSON Schema file]()
-	* [Verifying an entity with a JSON Schema remote]()
-	* [Verifying an entity with a JSON Schema remote (simplified mode)]()
-* [Real Example Case]()
+	* [Verifying an entity with a JSON Schema file](#verifying-an-entity-with-a-JSON-Schema-file)
+	* [Verifying an entity with a remote JSON Schema](#verifying-and-entity-with-a-remote-JSON-Schema)
+	* [Verifying an entity with a remote JSON Schema (simplified mode)](#verifying-an-entity-with-a-remote-JSON-Schema-(simplified-mode))
+* [Real Example Case](#real-example-case)
+* [Types of errors returned by the Data Models Schemas Analyzer](#types-of-errors-returned-by-the-data-models-schemas-analyzer)
 
 ## Importing JSON Schemas.
 There are two ways to import a JSON schema with the ngsi-parser module, from an external file or from a remote repository.
 
 ### Importing a JSON Schema from a remote repository.
-When you want to import a JSON schema from a remote repository, you can use the`setModel()` function. This function receives as parameter a JSON object with the URL of the JSON Schema. The next example shows the `setModel()` function which receives as parameter the JSON with URL of the JSON schema.
-Example:
+When you want to import a JSON schema from a remote repository, you can use the `setModel()` function. This function receives as parameter a JSON object with the URL of the JSON Schema. The next fragment of code shows an example of the `setModel()` function.
+
 ```javascript
 	var ngsi = require('ngsi-parser');
 	ngsi.setModel({
 		myRemoteSchema : 'https://yourdatamodels.com/myRemote'
 	});
 ```
 ### Importing a JSON Schema from an external JSON File.
-When you want to import a JSON schema from an external JSON schema file, you must import the external JSON schema file in the main file of the project, and store the schema in a variable. The variable of the JSON schema imported is used in the `setModel()` function. You must specify as parameter of the `setModel()` function , the JSON object that contains the variable of the JSON schema imported. The next example shows the JSON Schema imported in the project, called `mySchema.json`. This schema is stored in a variable called `mySchemaImported` and used in the `setModel()` function.
-Example:
+When you want to import a JSON schema from a JSON file, you must import this file in the main file of the project, and store the schema in a variable. The variable of the JSON schema imported is used in the `setModel()` function. For example, in the next fragment of code is imported the file `mySchema.json and save it in the variable `mySchemaImported, then, this variable is used in the `setModel()`function. 
```

Comment:
wrong place for closing tick (`), 
"...and save it..." may be "...and saved/stored in the variable..."